composer.json

```diff
@@ -75,6 +81,7 @@
         "symfony/css-selector": "^5.4 || ^6.0",
         "symfony/dependency-injection": "^5.4 || ^6.0",
         "symfony/deprecation-contracts": "^2.1 || ^3.0",
+        "symfony/doctrine-bridge": "^5.4 || ^6.0",
```

Comment:
Else Symfony 7 is installed which errors with:

<img width="1274" alt="Bildschirmfoto 2024-02-19 um 18 17 32" src="https://github.com/sulu/sulu/assets/1698337/fdc9960e-de50-4188-9675-ecd84ed5925d">

Reply:
you will eventually want to figure out why that happens ;-) but agree to not mix it into this checks.